src/main/commandServer/settingsValidator.ts

```diff
@@ -0,0 +1,185 @@
+type validationFunc = (desiredValue: string|boolean|Record<string, any>, errors: string[], fqname: string) => boolean;
```

Comment:
We should just drop this type definition: _none_ of the validation functions actually implement it.

Reply:
Sure. The actual signature changed and typescript didn't complain, so I'm not sold on the benefit of typescript with dispatch tables.